recipe/meta.yaml

```
@@ -8,22 +8,23 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sf-hamilton-{{ version }}.tar.gz
-  sha256: 91f71fa81e37efb156ae8901c4b7fc073f794de9ebe7ff8a59db5943de52c0eb
+  sha256: 9dda6ed13c829cedda02eda0713f17995ea23a8511f7a32df61d9744c249ee60
 
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True #[py<36]
```

```suggestion
  skip: True  # [py<36]
```
Generally, we put a selector `#` after two whitespaces at the end of the line